:memo: Add persistent Redis to standard (#2200)
platformsh/platformsh-docs: docs/src/configuration/services/redis.md

Versions 3.0 and higher support up to 64 different databases per instance of the service, but Redis 2.8 is configured to support only a single database.
{{< /note >}}

## Service types

There are two types of the Redis service depending on what you want to do with it.

### Ephemeral Redis

> The ephemeral Redis service is configured to serve as an LRU cache; its storage isn't persistent.

Reviewer: Since LRU is not the only type of behavior that can be expected (LFU, ...), we could also change the sentence to:
```suggestion
The ephemeral Redis service is configured to serve as a cache; its storage isn't persistent.
```
and remove:
> It isn't suitable for use except as a disposable cache.

It isn't suitable for use except as a disposable cache.

Data in an ephemeral Redis instance is stored only in memory and thus requires no disk space.
When the service hits its memory limit,

> it automatically evicts old cache items according to the [configured eviction rule](#eviction-policy) to make room for new ones.

Reviewer: ```suggestion
it automatically evicts cache items according to the [configured eviction rule](#eviction-policy) to make room for new ones.
```
